infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -190,26 +220,23 @@ public void shutdown() {
     }
 
     @Override
-    public boolean isServerReadyToHandleMsg(CorfuMsg msg) {
-        if (getState() != ServerState.READY){
+    public boolean isServerReadyToHandleMsg(RequestMsg request) {
+        if (getState() != ServerState.READY) {
             return false;
         }
 
-        if ((sequencerEpoch != serverContext.getServerEpoch())
-                && (!msg.getMsgType().equals(CorfuMsgType.BOOTSTRAP_SEQUENCER))) {
+        if ((sequencerEpoch != serverContext.getServerEpoch()) &&
+                (!request.getPayload().getPayloadCase()
+                        .equals(PayloadCase.BOOTSTRAP_SEQUENCER_REQUEST))) {
+
             log.warn("Rejecting msg at sequencer : sequencerStateEpoch:{}, serverEpoch:{}, "
-                    + "msg:{}", sequencerEpoch, serverContext.getServerEpoch(), msg);
+                            + "header:{}", sequencerEpoch, serverContext.getServerEpoch(),
+                    request.getHeader());
```

Comment:
Can we use `TextFormat.shortDebugString` here and in other places where we log a Protobuf message?

Reply:
Done, thanks!